src/breadcrumbs/index.js

```diff
@@ -13,7 +13,7 @@ class Breadcrumbs extends React.Component {
 
   render() {
     return (
-      <div className="ui breadcrumb">
+      <div className="ui small breadcrumb">
```

Comment:
Generally, when needing to change the size class of a component, it should be added as a prop so it's configurable later.